From a3c28f7756df6c80bba87ffa8c7e53aa48a0e58d Mon Sep 17 00:00:00 2001
From: Elimar Riesebieter <riesebie@lxtec.de>
Date: Sun, 8 Oct 2017 15:24:23 +0200
Subject: [PATCH] Adjusted NeoMutt version in auto.def

---
 auto.def | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: neomutt/auto.def
===================================================================
--- neomutt.orig/auto.def
+++ neomutt/auto.def
@@ -14,7 +14,7 @@ use system cc cc-lib mutt-gettext mutt-i
 ###############################################################################
 # Names and versions
 define PACKAGE          "neomutt"
-define PACKAGE_VERSION  "20180622"
+define PACKAGE_VERSION  "20180706-lxtec"
 define BUGS_ADDRESS     "neomutt-devel@neomutt.org"
 
 # Subdirectories that contain additional Makefile.autosetup files
