From 6e2bfa78bcf3dec2aaef6826084d762b793aafff Mon Sep 17 00:00:00 2001
From: Pietro Cerutti <gahr@gahr.ch>
Date: Mon, 9 Oct 2017 09:59:47 +0000
Subject: [PATCH 1/2] If set, use -D_FILE_OFFSET_BITS=64 when checking for
 GPGMe

Issue #826
---
 auto.def | 15 +++++++++++----
 1 file changed, 11 insertions(+), 4 deletions(-)

diff --git a/auto.def b/auto.def
index 95bb5000..4082a832 100644
--- a/auto.def
+++ b/auto.def
@@ -225,6 +225,7 @@ if {1} {
   cc-with [list -cflags [get-define CFLAGS]]
 
   # Large file support
+  cc-check-lfs
   cc-with {-includes sys/types.h} {
     if {[cc-check-sizeof off_t] eq {8}} {
       define OFF_T_FMT {"%" PRId64}
@@ -319,11 +320,17 @@ if {[get-define want-full-doc]} {define MAKEDOC_FULL}
 ###############################################################################
 # GPGME
 if {[get-define want-gpgme]} {
-  if {![check-inc-and-lib gpgme [opt-val with-gpgme $prefix] \
-                          gpgme.h gpgme_new gpgme]} {
-    user-error "Unable to find GPGME"
+  set gpgme_cflags {}
+  if {[is-defined _FILE_OFFSET_BITS]} {
+    set gpgme_cflags -D_FILE_OFFSET_BITS=[get-define _FILE_OFFSET_BITS]
+  }
+  cc-with [list -cflags $gpgme_cflags] {
+    if {![check-inc-and-lib gpgme [opt-val with-gpgme $prefix] \
+                            gpgme.h gpgme_new gpgme]} {
+      user-error "Unable to find GPGME"
+    }
+    cc-check-functions gpgme_op_export_keys
   }
-  cc-check-functions gpgme_op_export_keys
   define CRYPT_BACKEND_GPGME
 }
 
-- 
2.14.2

