From 06893a394d578ecf960691b5b17753366edbbac2 Mon Sep 17 00:00:00 2001
From: Pietro Cerutti <gahr@gahr.ch>
Date: Mon, 9 Oct 2017 13:36:31 +0000
Subject: [PATCH 2/2] Let OFF_T_FMT be "%" PRId64 whenever LFS is there

Issue #826
---
 auto.def | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/auto.def b/auto.def
index 4082a832..47f80cdf 100644
--- a/auto.def
+++ b/auto.def
@@ -225,15 +225,12 @@ if {1} {
   cc-with [list -cflags [get-define CFLAGS]]
 
   # Large file support
-  cc-check-lfs
-  cc-with {-includes sys/types.h} {
-    if {[cc-check-sizeof off_t] eq {8}} {
-      define OFF_T_FMT {"%" PRId64}
-    } else {
-      define OFF_T_FMT {"%" PRId32}
-    }
-    define LOFF_T off_t
+  if {[cc-check-lfs]} {
+    define OFF_T_FMT {"%" PRId64}
+  } else {
+    define OFF_T_FMT {"%" PRId32}
   }
+  define LOFF_T off_t
 
   # Let's always use volatile for sig_atomic_t
   cc-check-includes signal.h
-- 
2.14.2

